_rules/SVG-image-has-name.md

```diff
@@ -0,0 +1,143 @@
+---
+name: SVG image has accessible name
+description: |
+  This rule checks that each SVG image element that is included in the accessibility tree, has an accessible name.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any [SVG](https://www.w3.org/2000/svg) element with the [semantic role](#semantic-role) of `img` or `graphics-document`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element has a an accessible name.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+Browser and assitive technology support for SVG `title` and `desc` elements is currently inconsistent. Using WAI ARIA in combination with the `img` role for non-decorative `svg` elements significantly improves accessibility brwoser support.
```

Comment:
Typo at the the end 
"significantly improves accessibility brwoser support." > "significantly improves accessibility br**ow**ser support."